libraries/SimpleVocab/Controller/Plugin/SelectFilter.php

```diff
@@ -22,8 +22,10 @@ class SimpleVocab_Controller_Plugin_SelectFilter extends Zend_Controller_Plugin_
     protected $_defaultRoutes = array(
         array('module' => 'default', 'controller' => 'items', 
               'actions' => array('add', 'edit', 'change-type')), 
-        array('module' => 'default', 'controller' => 'elements', 
-              'actions' => array('element-form')), 
+        array('module' => 'default', 'controller' => 'files',
+              'actions' => array('add', 'edit', 'change-type')),
```

Comment:
The "change-type" action is related to Item Types, so it doesn't exist for Files